2_process/src/process_data.R

```diff
@@ -0,0 +1,22 @@
+############## Process data  ######################
+
+
+load_data <- function(mendota_file) {
+  read_csv(mendota_file, col_types = 'iccd') %>%
+    filter(str_detect(exper_id, 'similar_[0-9]+')) %>%
+    mutate(col = case_when(
+      model_type == 'pb' ~ '#1b9e77',
+      model_type == 'dl' ~'#d95f02',
+      model_type == 'pgdl' ~ '#7570b3'
+    ), pch = case_when(
+      model_type == 'pb' ~ 21,
+      model_type == 'dl' ~ 22,
+      model_type == 'pgdl' ~ 23
+    ), n_prof = as.numeric(str_extract(exper_id, '[0-9]+')))
+  
+}
+
+summary_model <- function (mendota_file) {
```

Comment:
The `project_output_dir` variable gets used within this `summary_model` function but is not declared as an argument - it should be, because in general your functions should not use anything from outside the function that's not declared as an argument.

Rather than simply creating and using an argument called `project_output_dir` here, though, let's go one step further and make the whole filepath into an argument that you pass in. That way the name of the file that gets written out by this function will be clearly visible at the top level of your project, in REMAKE.R.